unic/ucd/tests/category_tests.rs

```diff
@@ -40,3 +144,67 @@ fn test_bidi_nsm_against_gen_cat() {
         }
     }
 }
+
+/// `Bidi_Class=BN := General_Category in { Cc (Control), Co (PrivateUse) }`
+///
+/// <http://www.unicode.org/reports/tr9/#BN>
+#[test]
+fn test_bidi_bn_against_gen_cat() {
+    for cp in iter_all_chars() {
+        // Every BoundaryNeutral must satisfy GC::is_other()
+        if BidiClass::of(cp) == BidiClass::BoundaryNeutral {
+            assert!(GC::of(cp).is_other());
+        }
+
+        // Every !GC::is_other() must not be an BoundaryNeutral
+        if !GC::of(cp).is_other() {
+            assert_ne!(BidiClass::of(cp), BidiClass::BoundaryNeutral);
```

Comment:
I think this assertion fails if and only if the one above fails. If so, we can drop it.